client/core/core.go

```diff
@@ -1765,6 +1779,48 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 	return nil
 }
 
+// createSeededWallet initializes a seeded wallet with an asset-specific seed
+// derived deterministically from the app seed and a random password. The
+// password is returned for encrypting and storing.
+func (c *Core) createSeededWallet(assetID uint32, crypter encrypt.Crypter, form *WalletForm) ([]byte, error) {
+	creds := c.creds()
+	if creds == nil {
+		return nil, fmt.Errorf("no v2 credentials stored")
+	}
+
+	root, err := crypter.Decrypt(creds.EncSeed)
+	if err != nil {
+		return nil, fmt.Errorf("seed decryption error: %w", err)
+	}
+
+	c.log.Infof("Initializing a built-in %s wallet", unbip(assetID))
```

Comment:
Can we use this grammar (built-in wallet) generally instead of seeded wallet? The latter sounds confusing to me and I expect it also won't make immediate sense to a regular user, but no hard preferences. I do prefer builtin because all wallets are seeded really, aren't they?

Regardless of which terminology we stick with, I'll recommend consistent messaging in logs and errors, e.g. in `core.CreateWallet`, I saw `external password incompatible with seeded wallet`.

Reply:
I think built-in makes more sense too. All wallets are seeded.